deploy/operator-openshift.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: jaeger-operator
-          image: jaegertracing/jaeger-operator:1.7.0
+          image: jaegertracing/jaeger-operator:master
```

Comment:
I'd rather cut a new release now, 1.7.x. We could actually do a new release every week so that our examples won't be ahead for too long.

Reply:
That is also fine with me.